packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.h

```diff
@@ -4,4 +4,8 @@
 #import <UMCore/UMModuleRegistryConsumer.h>
 
 @interface EXWebBrowser : UMExportedModule <UMModuleRegistryConsumer>
+
++ (UIColor *)convertHexColorString:(NSString *)stringToConvert;
```

Comment:
I think we don't need to expose those two methods outside of this class 🤔

Reply:
Has this been resolved by accident or do you think we should expose these methods?

Reply:
By accident, sorry :)